app/components/service/serviceController.js

```diff
@@ -269,6 +272,10 @@ function ($q, $scope, $stateParams, $state, $location, $anchorScroll, ServiceSer
       $scope.service = service;
       ControllerDataPipeline.setAccessControlData('service', $stateParams.id, service.ResourceControl);
       originalService = angular.copy(service);
+   
+      if(page_anchor) {
```

Comment:
Then just call `$anchorScroll()` here instead of that condition, and it should automatically scroll the hash when the page is refreshed.